fuzz/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ fuzzer(privkey)
 fuzzer(cert)
 fuzzer(spki)
 fuzzer(pkcs8)
```

Comment:
Not sure if we should rename this to `pkcs8_v1`, but the current version does correspond with `EVP_marshal_private_key`/`EVP_marshal_private_key_v2`

Reply:
I'm going to bias towards leaving it the same for now to stay consistent with BoringSSL upstream. Can revisit later if needed.